clientupdate/clientupdate.go

```diff
@@ -936,12 +936,18 @@ func (up *Updater) updateFreeBSD() (err error) {
 		return nil
 	}
 
-	cmd := exec.Command("pkg", "upgrade", "tailscale")
+	cmd := exec.Command("pkg", "upgrade", "-y", "tailscale")
 	cmd.Stdout = up.Stdout
 	cmd.Stderr = up.Stderr
 	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("failed tailscale update using pkg: %w", err)
 	}
+
+	// pkg does not automatically restart services after upgrade.
+	out, err = exec.Command("service", "tailscaled", "restart").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to restart tailscaled after update: %w, output: %q", err, out)
```

Comment:
It might be better _not_ to quote the output, since it's likely to be a bunch of lines anyway. Perhaps `… output:\n%s", …` ? (YMMV)

Reply:
Yeah, that's fine. In the logs it will be quoted anyway, since the logs are wrapped in JSON objects.

Reply:
Ah, neat! Updated everywhere in the file.